team/bundles/org.eclipse.core.net/src/org/eclipse/core/net/proxy/win32/ProxyBypass.java

```diff
@@ -12,7 +12,7 @@
  * 	compeople AG (Stefan Liebig) - initial API and implementation
  *  IBM Corporation - Add proxy providers layer on the top of ProxyManager (bug 255616)
  *******************************************************************************/
-package org.eclipse.core.internal.net.proxy.win32.winhttp;
+package org.eclipse.core.net.proxy.win32;
```

Comment:
Make it internal package. Definetely not smth we want anyone to consider API.

Reply:
Done.